lib/utils.js

```diff
@@ -452,6 +456,15 @@ class Literal extends SequelizeMethod {
 }
 exports.Literal = Literal;
 
+class Composition extends SequelizeMethod {
```

Comment:
This feels like a workaround, can we not have the base `Composition` class inherit from a sequelize method?

Reply:
I've rename this to `CompositionMethod` to make the code clearer..

Regarding this question, I don't think `Composition` can inherit from `SequelizeMethod`. `SequelizeMethod` is an intermediate object that requires processing(`handleSequelizeMethod`) to be converted to a part of the query, which is a Composition.